File: src/Omnia.ProcessManagement.Web/client/fx/models/data/drawingdefinitions/CanvasDefinition.ts
import { DrawingShape } from './DrawingShape';

export interface ICanvasDefinition {
    backgroundImageUrl?: string;
    width: number;
    height: number;
    gridX?: number;
    gridY?: number;
}

export interface CanvasDefinition extends ICanvasDefinition {
    drawingShapes: Array<DrawingShape>;
}

File: src/Omnia.ProcessManagement.Models/CanvasDefinitions/CanvasDefinition.cs
﻿using Omnia.ProcessManagement.Models.Images;
using System;
using System.Collections.Generic;
using System.Text;

namespace Omnia.ProcessManagement.Models.CanvasDefinitions
{
    public class CanvasDefinition
    {
        public ImageRef BackgroundImageRef { get; set; }
        public int Width { get; set; }
        public int Height { get; set; }
        public int GridX { get; set; }
        public int GridY { get; set; }
        public List<DrawingShape> DrawingShapes { get; set; }
    }
}
